client/devdocs/style.scss

```diff
@@ -199,3 +199,28 @@
 		}
 	}
 }
+
+.docs__post-schedule {
+	@include breakpoint( ">480px" ) {
+		display: flex;
+		justify-content: space-between;
+	}
+}
+
+.docs__post-schedule-card {
+	display: inline-block;
```

Comment:
With the `display: flex` the `display: inline-block` shouldn't be necessary anymore, I think?